client/app/components/datagrid/DataGrid.tsx

```diff
@@ -76,7 +114,73 @@ const DataGrid: React.FC<Props> = ({ rows, columns, cntxt }) => {
 
   return (
     <div style={{ height: "auto", width: "100%" }}>
-      <MuiGrid rows={rows} columns={customColumns} disableVirtualization />
+      <MuiGrid
+        rows={rows}
+        columns={customColumns}
+        showCellVerticalBorder
+        components={{
+          ColumnSortedAscendingIcon: AscendingIcon,
+          ColumnSortedDescendingIcon: DescendingIcon,
+          ColumnUnsortedIcon: SortIcon,
+        }}
+        sx={{
+          "& .MuiSvgIcon-root": {
+            color: "white",
+          },
+          "& .MuiDataGrid-columnHeaderDraggableContainer": {
+            minWidth: "100%",
+          },
+          "& .MuiDataGrid-columnHeader": {
+            border: "1px white solid",
+            borderBottom: "none",
+            borderTop: "none",
+            color: "white",
+            backgroundColor: BC_GOV_BACKGROUND_COLOR_BLUE,
+            fontWeight: "bold",
+            justifyContent: "center",
+          },
+
+          "& .MuiDataGrid-columnHeader:first-child": {
+            borderLeft: "none",
+          },
+          "& .MuiDataGrid-columnHeader:last-child": {
+            borderRight: "none",
+          },
+          "& .MuiDataGrid-columnHeaderTitleContainer": {
+            justifyContent: "space-between",
+          },
+          "& .MuiDataGrid-columnHeaderTitleContainerContent": {
+            flex: 1,
+          },
+          "& .MuiDataGrid-columnHeaderTitle": {
+            fontWeight: "bold",
+            whiteSpace: "pre-line",
+            lineHeight: "normal",
+            textAlign: "center",
+            width: "100%",
+            minWidth: "100%",
+          },
+          "& .MuiDataGrid-columnSeparator": {
+            display: "none",
+          },
+          "& .MuiDataGrid-cell": {
+            justifyContent: "center",
+            textAlign: "center",
+          },
+          ".MuiDataGrid-iconButtonContainer": {
+            visibility: "visible !important",
+            minWidth: "20px",
+          },
+          "& .MuiDataGrid-sortIcon": {
+            opacity: "inherit !important",
+          },
+          // We may want tor eenable this functionality though it's not in the design
+          ".MuiDataGrid-menuIcon": {
+            display: "none",
+          },
```

Comment:
This is a bit of regression in functionality as this hides the menu with the 3 dots that has advanced filtering. Though it wasn't in the design.

Flagging incase whoever reviews this thinks it should be added back.